plasma_framework/docs/design/add-new-tx-type-check-lists.md

```diff
@@ -0,0 +1,9 @@
+# Check List For Adding New Transaction Types
+
+This document provides some check lists that whenever we are designing a new transaction type. Since there are some known restriction of the Plasma Framework design, please make sure that new tx type and exit game does not break the global assumptions and limitations.
+
+1. Aside from deposit transactions, all transactions should be unique. This promises the `OutputId` to be unique as `OutputId` uniqueness is a global assumption placed to `PlasmaFramework`.
```

Comment:
What does `uniqueness` of a transaction mean? Do you mean the type or the two txs of the same type to look the same? I would also write sth about how to design a new `output id`.